content/docs/concepts/staking/_index.md

```diff
@@ -198,19 +200,32 @@ For example, an unbonding scenario with an epoch period of 30 blocks, an unbondi
 
 ## Staking transitions
 
-Staking transitions are the application of changes to stake bonded to validators. Newton stake token can be in [three states](/concepts/protocol-assets/newton/):
- - The default state of `unbonded`.
- - The locked state of `bonded`.
- - An intermediate state of `unbonding` when it is locked pending redemption.
+Staking transitions are changes to stake bonded to validators caused by:
+
+- stake bonding and unbonding operations submitted by stake delegators
+- [slashing](/concepts/staking/#slashing) penalties applied by protocol for proven validator faults whilst a member of the consensus committee. 
```

Comment:
not sure I would consider slashing a staking transition (even though it reduces the bonded stake)

Reply:
I see the point.  I've re-read this and agree it's inconsistent with other sections in the Concept page as well.

Ive reworded to remove this and edit - see commit 9652dc7fe7d62791ddd7958be06083fe8f9a320e